setup.py

```diff
@@ -47,5 +47,5 @@ def readme():
                      "wordcloud", "textblob", "plotly>=4.4.1", "cufflinks>=0.17.0", "umap-learn",
                      "lightgbm>=2.3.1", "pyLDAvis", "gensim", "spacy", "nltk", "mlxtend",
                      "pyod", "catboost>=0.20.2", "pandas-profiling>=2.3.0", "kmodes>=0.10.1",
-                     "datefinder>=0.7.0", "datetime", "DateTime>=4.3", "awscli"]
+                     "datefinder>=0.7.0", "datetime", "DateTime>=4.3", "awscli", "category_encoders>=2.0.0"]
```

Comment:
Em...

pycaret wrote here https://github.com/pycaret/pycaret/pull/78#issuecomment-620763289 that they want everything to be within 1 file, and with this https://github.com/pycaret/pycaret/pull/81 everything is going to be in requirements.txt.

So if you revert this commit it will have no conflict with https://github.com/pycaret/pycaret/pull/81 and after that change requirements.txt instead, so that your requirement will be included, please :)

In any case, this commit makes missync between reqirements.txt and setup.py that should be resolved.

Reply:
I tried to revert but i couldn't

Reply:
Done